C/services/south-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -79,6 +79,13 @@ void *PluginInterfaceInit(const char *pluginName, const char * pluginPathName)
     PyList_Append(sysPath, PyUnicode_FromString((char *) southRootPath.c_str()));
     PyList_Append(sysPath, PyUnicode_FromString((char *) fledgePythonDir.c_str()));
 
+    // Set sys.argv for embedded Python 3.5
```

Comment:
Don't we need same fix for

C/services/notification-plugin-interfaces/python/python_plugin_interface.cpp ?

if yes, please add it

Reply:
@AmandeepArora I am assuming the answer to this is yes we should add this for the notifications, correct?

Reply:
Yes, adding.

Reply:
Done, please verify using some notification python plugin by importing tensorflow.

Reply:
@AmandeepArora please suggest something easy to test. Some startup example code.
I.e. plugin skeleton, instructions for installing tensorflow etc

I just have the notification plugins with Numpy and nothing else.

Reply:
You can install tensorflow using "pip install tensorflow"
And in python plugin, at top you can add "import tensorflow as tf" and in plugin_init, you can print "tf.__version__" using _LOGGER.info. That should suffice, earlier we were getting crash dump just at import time.

Reply:
It is "`tf.__version__`"